cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -219,6 +219,10 @@ include::../indexes/create-a-single-property-index-for-nodes.asciidoc[leveloffse
 
 include::../indexes/create-a-single-property-index-for-relationships.asciidoc[leveloffset=+1]
 
+[NOTE]
+Relationship property indexes is a feature under development. If you want to try it out you need to set `unsupported.dbms.enable_relationship_property_indexes` setting in the _neo4j.conf_ file.
+Please note that no guarantees are made about functionality until the feature is complete.
+
```

Comment:
Currently reviewing this message with PM team. 

With this being "preview" docs prior to release, I don't think there is any necessity to declare this as work in progress. If this feature is only surfaced in Aura, then we should add this comment to the internal, unsupported documentation here: https://sites.google.com/neotechnology.com/dbms-internal-documentation/internal-configuration?authuser=1

Reply:
Okay, we don't actually want anybody trying it out yet. We just didn't want to remove the documentation just to have to put it back again later.

Reply:
Perhaps it might be best to add this in as comments then, and add something to the internal docs?